addons/hr_skills/static/src/scss/hr_employee.scss

```diff
@@ -10,3 +10,7 @@
         overflow-x: hidden!important;
     }
 }
+
+th[data-name="level_progress"] div span {
```

Comment:
Should be scoped to `.o_field_skills_one2many` to prevent any side effect if someone decides to add a `level_progress` field somewhere else

Reply:
You can add a `class="o_hr_skill_type_form"` to the `<form>`